src/domain.rs

```diff
@@ -306,6 +373,37 @@ fn best_fft<E: Engine + gpu::GpuEngine, T: Group<E::Fr>>(
     }
 }
 
+fn best_fft3<E: Engine + gpu::GpuEngine, T: Group<E::Fr>>(
+    worker: &Worker,
+    kern: &mut Option<gpu::LockedFFTKernel<E>>,
+    a0: &mut [T],
+    a1: &mut [T],
+    a2: &mut [T],
```

Comment:
Is there a reason why this isn't also just an array, like `omegas` and `log_ns` is?

Reply:
I am not a fan of puting mutable references in lists (there is no good reason, just a personal preference). This could probably be abstraced further, and just made to work with `n` elements instead of fixed `3`.

Reply:
switched to a more general `_many` interface